scripts/actions/spells/summoning/cait_sith.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Spell: Diabolos
+-- Summons Diabolos to fight by your side
```

Comment:
Nitpick: Wrong spell comment and description

Reply:
Oops, good catch.

Guess you can see what I copied from :)